rust-runtime/aws-smithy-runtime/src/client/orchestrator.rs

```diff
@@ -139,6 +139,11 @@ pub async fn invoke_with_stop_point(
     runtime_plugins: &RuntimePlugins,
     stop_point: StopPoint,
 ) -> Result<InterceptorContext, SdkError<Error, HttpResponse>> {
+    let mut random_bytes = [0u8; 16];
+    fastrand::Rng::default().fill(&mut random_bytes);
+    // This is for internal use only, serving to correlate logs to a single operation.
+    // It always emits a random UUID per client and is currently not configurable.
+    let id = uuid::Builder::from_random_bytes(random_bytes).into_uuid();
```

Comment:
Any reason to not use something like what we do for [waiters](https://github.com/smithy-lang/smithy-rs/blob/5bd44f26973532e9d16e0d1b175e72e40e75f6ca/rust-runtime/aws-smithy-runtime/src/client/waiters.rs#L280)? 

UUID will obviously reduce the chance of re-use of a single ID but I would imagine in practice logs being separated enough to know they are different. Possible advantage being a shorter log line.

Reply:
I had looked at that in waiters and was on the fence, actually. But you mentioned a shorter log line, which makes me go for that in waiters. I believe it helps reduce cognitive cost when looking through logs.

Reply:
Updated in 3fee07f